tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchCMISTests.java

```diff
@@ -165,6 +169,15 @@ public void matchDocumentName()
         searchQueryService.expectResultsFromQuery(query, user1, FILE_0_NAME);
     }
 
+    @TestRail (description = "Check ORDER BY syntax works as expected", section = TestGroup.SEARCH, executionType = ExecutionType.REGRESSION)
+    @Test (groups = TestGroup.SEARCH)
+    public void checkOrderByAscSyntax()
+    {
+        List<String> expectedList = orderNames(FILE_0_NAME, FILE_1_NAME, FILE_2_NAME);
+        SearchRequest query = req("cmis", "SELECT * FROM cmis:document WHERE cmis:name IN('" + FILE_0_NAME + "','" + FILE_1_NAME + "','" + FILE_2_NAME + "') ORDER BY cmis:name ASC");
```

Comment:
I think it would make sense to merge this piece of work first and then expand by adding tests for ordering of multiple fields. This build should now be green after [https://github.com/Alfresco/alfresco-enterprise-repo/pull/840](url) has been merged